README.md

```diff
@@ -7,61 +7,198 @@
+    print("Zzzz 💤 😴")
+}
+
+var classIsHard = false
+
+if classIsHard {
+    studyMore()
+} else {   
+    catchUpOnSleep()
+}
 
-if myHeight > 60 {
-    print("You're free to go on the ride, enjoy!")
+// prints "Zzzz 💤 😴"
+````
+Similar to above, but we added another function named catchUpOnSleep() which prints 'Zzzz 💤 😴' when called on. Then we are creating the classIsHard variable but setting it's initial value to false. Next we've created an if-else statement which should read like a sentence. If the class is hard, then we should study more.. if not, then we should catch up on some sleep. As you can see, what prints to console is 'Zzzz 💤 😴' which means studyMore() does not get called, we do not enter those { } curly braces, but we do enter the { } braces after the else statement because the first if statement evaluated to false. If it evaluated to true, then 'Study 📖' would have printed to the console and the catchUpOnSleep() function would never have gotten called.
+
+Are you begining to see the pattern?  Let's look at another one, maybe something more like an actual program, like "if the file exists then open it, otherwise tell the user it can't be found".  Let's see what it would look like in code:
+
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == true {
+	openFileNamed("testfile")
 } else {
-    print("I'm sorry, you're too short to go on this ride today.")
+	reportError("testfile not found")
 }
+````
+
+Every Conditional effectively has both an ````if```` and an ````else```` part but depending upon how we style our code we can leave out the explicit ````else```` (which the compiler interprets as "if the condition is false then don't do anything").  For example, we could re-write the code above as follows:
 
-// prints "I'm sorry, you're too short to go on this ride today."
-```
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == false {
+	reportError("testfile not found")
+	return
+}
+openFileNamed("testfile")
+````
 
-* Create/chain multiple *if* statements together to consider additional clauses:
+Can you see how they'e variations of the same thing?  Let's look at another example.
 
-```swift
-var temperatureInFahrenheit = 100
+````Swift
+var rainingOutside = false
 
-if temperatureInFahrenheit < 32 {
-    print("Snow! 🌨")
-} else if temperatureInFahrenheit >= 90 {
-    print("Very hot! ☀️")
+if rainingOutside == true {
+	takeUmbrella()
 } else {
-    print("Just right. 🌻")
+	// don't worry about taking the umbrella
+	// NB no code is being excuted here, only comments!
+}
+goForAWalk()
+````
+is the same as...
+
+````Swift
+func takeUmbrella() {
+    // grab umbrella
+    print("I have my umbrella! ☂️")
 }
-```
 
 
+}
+goForAWalk()
+````
 
-## What the student can do at this point 
+Our ````if```` doesn't have to be followed by a Boolean expression; it can also take Boolean variable or constant, or a function which returns a Boolean.  Here's what that would look like in code:
```

Comment:
Spelling fix, should read as:

"followed by a Boolean expression; it can also take **a** Boolean variable"